deploy/charts/rook-ceph/values.yaml

```diff
@@ -673,3 +674,55 @@ monitoring:
   # -- Enable monitoring. Requires Prometheus to be pre-installed.
   # Enabling will also create RBAC rules to allow Operator to create ServiceMonitors
   enabled: false
+
+ceph-csi-operator:
```

Comment:
@Madhu-1 When I tried to use the `ceph-csi-operator.enabled` in a helm condition, I get an error because it does not like the dash `-` as a character in the var name that we are checking. For example, in templates/configmap.yaml line 34 :
```yaml
  ROOK_USE_CSI_OPERATOR: {{ .Values.csi.ceph-csi-operator | quote }}
```
Results in this error:
```
Error: UPGRADE FAILED: parse error at (rook-ceph/templates/configmap.yaml:34): bad character U+002D '-'
```
Can we rename the `ceph-csi-operator` settings in the csi operator chart so it doesn't have dashes? Then we could use it in rook conditions.

Reply:
Actually we don't need this field, we can use `rook.csi.rookUseCSIOperator` value